OPHD/States/MapViewStateHelper.cpp

```diff
@@ -290,6 +290,8 @@ bool selfSustained(StructureID id)
  */
 bool inCommRange(NAS2D::Point<int> position)
 {
+
+
 	const auto maxCCRangeSquared = constants::ROBOT_COM_RANGE * constants::ROBOT_COM_RANGE;
 	const auto ccDistance = position - ccLocation();
 	if (ccDistance.lengthSquared() <= maxCCRangeSquared)
```

Comment:
Spurious extra blank lines added here.

Edit: Removed in later commit.

Reply:
Yeah, I accidentally added those.